ArrtSource/Storage/UI/StorageBrowserWidget.cpp

```diff
@@ -333,7 +333,10 @@ void StorageBrowserWidget::UploadItems(const QStringList& files)
         return;
     }
 
-    m_storageAccount->GetFileUploader()->UploadFilesAsync(rootDirectory, toUpload, GetSelectedContainer(), dstFolder);
+    if (auto file_uploader = m_storageAccount->GetFileUploader(); file_uploader != nullptr)
+    {
+        m_storageAccount->GetFileUploader()->UploadFilesAsync(rootDirectory, toUpload, GetSelectedContainer(), dstFolder);
```

Comment:
nit: could be
file_uploader->UploadFilesAsync

Reply:
thanks, fixed!